handlers/handlers.go

```diff
@@ -20,13 +24,66 @@ var reCaptchaSiteKey = os.Getenv("RECAPTCHA_SITE_KEY")
 var reCaptcha = recaptcha.R{
 	Secret: os.Getenv("RECAPTCHA_SECRET_KEY"),
 }
+var m = minify.New()
+
+func compileTemplates(templatePaths ...string) (*template.Template, error) {
+	var tmpl *template.Template
+	for _, templatePath := range templatePaths {
+		name := filepath.Base(templatePath)
+		if tmpl == nil {
+			tmpl = template.New(name)
+		} else {
+			tmpl = tmpl.New(name)
+		}
+
+		b, err := ioutil.ReadFile(templatePath)
+		if err != nil {
+			return nil, err
+		}
+
+		mb, err := m.Bytes("text/html", b)
+		if err != nil {
+			return nil, err
+		}
+		tmpl.Parse(string(mb))
+	}
+	return tmpl, nil
+}
+
+func minifyCSSFiles(templateDirectory string) {
+	cssFileDirectory := filepath.Join(templateDirectory, "../assets/css/")
+	cssFilePaths, _ := filepath.Glob(filepath.Join(cssFileDirectory, "*.css"))
+	for _, cssFilePath := range cssFilePaths {
+		if cssFilePath[len(cssFilePath)-8:] != ".min.css" {
+			continue
+		}
+		cssFile, err := ioutil.ReadFile(cssFilePath)
+		if err != nil {
+			panic(err)
+		}
+		cssFile, err = m.Bytes("text/css", cssFile)
+		if err != nil {
+			panic(err)
+		}
+		cssFilePathBase := filepath.Base(cssFilePath)
+		miniCSSFilePath := filepath.Join(cssFileDirectory, cssFilePathBase[:len(cssFilePathBase)-3]) + "min.css"
```

Comment:
Please replace

```
miniCSSFilePath := filepath.Join(cssFileDirectory, cssFilePathBase[:len(cssFilePathBase)-3]) + "min.css"
```

with

```
minCSSFilePath := strings.Replace(cssFilePath, ".css", ".min.css", 1)
```

(Note that _mini_ was renamed to _min_. After updating, check that it works)